main.py

```diff
@@ -13,19 +20,23 @@ def mean_reversion_decision(isin: str, x1: str = "d1"):
     :param x1: pass what type of data you want to retrieve (m1, h1 or d1)
     :return: returns whether you should buy (True) or sell (False), depending on MR criteria
     """
-    market_data = Instruments(
+    from_date = (datetime.now() - timedelta(days=7)).strftime('%Y-%m-%d')
+    market_data = client.market_data.ohlc.get(
         isin=isin,
-        x1=x1
-    ).get_market_data()
+        from_=from_date,
+        period=x1
+    )
     print(market_data)
-    d1_prices = market_data['results']
-    prices_close = [x["c"] for x in d1_prices]  # you can obviously change that to low, close or open
+
+    d1_prices = market_data.results
+    prices_close = [x.c for x in d1_prices]  # you can obviously change that to low, close or open
     mean_price = statistics.mean(prices_close)
     print(f'Mean Price: {mean_price}')
-    latest_close_price = Instruments(
+    latest_close_price = client.market_data.ohlc.get(
         isin=isin,
-        x1="m1"
-    ).get_latest_market_data()
+        from_='latest',
+        period="m1"
+    ).results[0].c
```

Comment:
work with "get" here as well to make it more bullet proof in case no data is available?